neutron/files/ocata/neutron-generic.conf.Debian

```diff
@@ -91,6 +91,9 @@ dhcp_lease_duration = 600
 
 # Domain to use for building the hostnames (string value)
 #dns_domain = openstacklocal
+{% if neutron.get('dns_domain') %}
```

Comment:
provide default False

Reply:
Done.  Pushing a update shortly.

Reply:
It is neutron.dns_domain because   ``neutron`` is neutron.gateway from the map.jinja in the context of this template: https://github.com/salt-formulas/salt-formula-neutron/blob/master/neutron/files/ocata/neutron-generic.conf.Debian#L2